tests/auth_tests/backend_alias.py

```diff
@@ -1,4 +1,4 @@
 # For testing that auth backends can be referenced using a convenience import
-from .test_auth_backends import ImportedModelBackend
+from .test_auth_backends import ImportedModelBackend, OtherImportedModelBackend
 
-__all__ = ['ImportedModelBackend']
+__all__ = ['ImportedModelBackend', 'OtherImportedModelBackend']
```

Comment:
Putting these new tests in with the "ImportedBackend" backend tests doesn't make sense as that previous bug is not really relevant here. I think it would be clearer to function directly if not too difficult.